packages/combobox/src/LionCombobox.js

```diff
@@ -12,13 +11,16 @@ import { LionListbox } from '@lion/listbox';
  * @typedef {import('@lion/listbox').LionOptions} LionOptions
  * @typedef {import('@lion/overlays/types/OverlayConfig').OverlayConfig} OverlayConfig
  * @typedef {import('@lion/core/types/SlotMixinTypes').SlotsMap} SlotsMap
+ * @typedef {import('@lion/form-core/types/choice-group/ChoiceInputMixinTypes').ChoiceInputHost} ChoiceInputHost
+ * @typedef {import('@lion/form-core/types/FormControlMixinTypes').FormControlHost} FormControlHost
  * @typedef {import('../types/SelectionDisplay').SelectionDisplay} SelectionDisplay
  */
 
 /**
  * LionCombobox: implements the wai-aria combobox design pattern and integrates it as a Lion
  * FormControl
  */
+// @ts-expect-error static properties are not compatible
```

Comment:
Type as `any`

Reply:
True, if I'm correct this was fixed in the types PR after this one

Reply:
See https://github.com/ing-bank/lion/pull/1328